deploy/docker/Dockerfile

```diff
@@ -3,19 +3,26 @@ FROM fluent/fluentd:v1.11.1-debian-1.0 AS builder
 # Use root account to use apt
 USER root
 
-# Build dependencies
-RUN buildDeps="sudo make gcc g++ libc-dev ruby-dev libsnappy-dev" \
- && apt-get update \
- && apt-get install -y --no-install-recommends $buildDeps
-
-COPY gems/fluent-plugin*.gem ./
+# Dependencies
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+       sudo \
```

Comment:
can we list these in alphabetical order?

Reply:
"yes we can"